serve.js

```diff
@@ -16,26 +17,18 @@ const args = require('minimist')(process.argv.slice(2));
 const compression = require('compression');
 const child_process = require('child_process');
 // -------------------------------------------
-
-if (args.hasOwnProperty('help') || args._.includes('help')) {
-    console.log(`Usage: ./serve.js [OPTION]...
-Starts the Aspine web server.
-
-Options:
-  --insecure  do not secure connections with TLS (HTTPS)
-  --help      display this help and exit
-`);
-    process.exit();
-}
+program
+    .version('2.6.1')
+    .option('-i, --insecure', 'do not secure connections with TLS (HTTPS)')
+    .option('-p, --port <number>', 'port to listen on', 8080)
+    .parse();
 
 // ------------ Web Server -------------------
-const port = 8080;
-
 const app = express();
-app.use(compression({ filter: (..._) => true }));
-app.listen(port, () => console.log(`Aspine listening on port ${port}!`));
+app.use(compression());
+app.listen(program.port, () => console.log(`Aspine listening on port ${program.port}!`));
```

Comment:
@notrodes Why did you remove the custom `filter` function? I had included it because `compression` by default does not compress all traffic and so feeding it a function that always returns `true` causes all traffic to be compressed.

Reply:
By the way, I think it should be fine to change the function from `(..._) => true` to the more concise `() => true`, but we should double-check that everything is being compressed.

Reply:
i checked and the default compresses based on what data type it is being used. I though that would be better than everything being compressed.

Reply:
I guess the default algorithm changed since I did this in #125, because now the difference in amount of data transferred is negligible between full compression and default compression. In that case, `compression()` is indeed probably better.